src/spans/awsSpan.ts

```diff
@@ -242,7 +244,7 @@ export const getEndFunctionSpan = (functionSpan: GenericSpan, handlerReturnValue
   const ended = new Date().getTime();
   let returnValue: any;
   try {
-    returnValue = payloadStringify(data);
+    returnValue = payloadStringify(clone(data));
```

Comment:
Maybe we should the `clone` inside `payloadStringify`? if we're concerned about it recursively cloning every subset of the object, let's make it:
```ts
const payloadStringify = payload => originalPayloadStringify(clone(data))
```
that way we won't have potential issue in otherexisting of future use cases of `payloadStringify`. WDYT?

Reply:
@harelmo-lumigo does make sense. However, in this approach, we will need to calculate the time impact. 
I propose to have it inside `payloadStringify` but only in the 
```
if (getSecretMaskingExactPath()) {}
```
section. As we know it is implementation `LUMIGO_SECRET_MASKING_EXACT_PATH`

Reply:
For now, let's not change it and keep it that way.
I'd suggest to add a ticket for considering the clone at a single point in the process later, as a separate change we can benchmark easily